modules/dnn/src/webnn/README.md

```diff
@@ -0,0 +1,10 @@
+## Build WebNN-native to enable native implementation for DNN_BACKEND_WEBNN  in modules/dnn
+
+
+### Build WebNN-native and set the environment variable
+
+ Refer to [WebNN's build instructions](https://github.com/webmachinelearning/webnn-native) to complete the build of WebNN-native (build the Release version). Set environment variable `WEBNN_NATIVE_DIR` to enable native DNN_BACKEND_WEBNN build: `export WEBNN_NATIVE_DIR=${PATH_TO_WebNN}`.
```

Comment:
Correspondingly, please require to set `WEBNN_NATIVE_DIR` to output directory of webnn-native build, e.g. `webnn-native/out/Release`.

Reply:
Thanks! I've added this instruction.